llmware/library.py

```diff
@@ -691,7 +1071,18 @@ def add_wiki(self, topic_list,target_results=10):
 
     def add_dialogs(self, input_folder=None):
 
-        """ Main method to add an AWS dialog transcript into a library """
+        """ Main method to add an AWS dialog transcript into a library 
```

Comment:
Delete space before you start the doc and add full stop.
```suggestion
        """Main method to add an AWS dialog transcript into a library.
```